pub/index.html

```diff
@@ -0,0 +1,168 @@
+<!doctype html>
+<html>
+  <head>
+    <title>south park blog</title>
+    <style>
+    body {
+      margin: 0px;
+      padding: 0px;
+    }
+    header {
+      background-image: url(http://www.importantstudio.com/ironyard/spbusstop.jpg);
+      margin: 0;
+      width: 1280px;
+      height: 350px;
+      margin: 0px;
+    }
+    .container {
+        width: 850px;
+        padding: 10px;
+        margin: 0 auto 0 auto;
+        height: 1000px;
+
+    }
+    aside {
+      width: 250px;
+      padding: 0 10px 0 10px;
+      float: left;
+
+    }
+    .logodiv {
+      width: 100px;
+      height: 100px;
+      position: relative;
+      top: -70px;
+      border-radius: 50%;
+      background: url(http://www.importantstudio.com/ironyard/sps.jpg) center center no-repeat;
+      border: 4px solid white;
+    }
```

Comment:
Personally, I would keep the logo image on the page as HTML rather than applying a background image in this case... Wrapping the image in a block-level element with `class="logo"` is a great way to convey the semantic meaning of that image to Google, for one.